tools/gearman_top.c

```diff
@@ -90,6 +91,9 @@ int main (int argc, char **argv) {
     }
     mod_gm_opt->debug_level = opt_verbose;
     mod_gm_opt->logmode     = GM_LOG_MODE_TOOLS;
+    if((env_server = getenv("CONFIG_GEARMAND_PORT")) != NULL) {
+        server_list[server_list_num++] = env_server;
+    }
     if(server_list_num == 0)
```

Comment:
could you move that in the if block here, so the env value will only be used if nothing specified on the command line?